web_search_autocomplete_prefetch/static/src/js/web_search_autocomplete_prefetch.js

```diff
@@ -83,6 +84,7 @@ openerp.web_search_autocomplete_prefetch = function(instance)
     });
 
     instance.web.search.AutoComplete.include({
+        keypress_timeout: 350,
```

Comment:
@legalsylvain already proposed this to the customer, they agreed, it's in https://github.com/OCA/web/pull/614/commits/08b3b724f042f3269b0ba60ce6ca343fd3e26a58